File: Java/src/main/java/com/gildedrose/item/Sulfuras.java
/* (C)2022 */
package com.gildedrose.item;

public class Sulfuras extends Item {
    public static final String SULFURAS = "Sulfuras, Hand of Ragnaros";
    public static final int QUALITY = 80;

    public Sulfuras(int sellIn) {
        super(SULFURAS, sellIn, QUALITY);
    }

    @Override
    public void updateQuality() {}

    @Override
    public void updateSellIn() {}
}

File: python/src/sulfuras.py
from python.components.gilded_rose import GildedRose, Item


class SulfurasLogic(GildedRose):
    def __init__(self, item: Item):
        super().__init__(item)

    def update_quality(self):
        # "Sulfuras" does not change in quality or sell_in
        return self.quality, self.sell_in